ui/templates/mitx-online-required.html

```diff
@@ -0,0 +1,32 @@
+{% extends "base.html" %}
+{% load i18n static %}
+
+{% block title %}{% trans "MITx MicroMasters" %}{% endblock %}
+{% block bodyclass %}signin-page{% endblock %}
+
+{% block content %}
+
+<div id="signin-page">
+{% include "header.html" %}
+
+<div class="main-content-wrapper">
+  <div class="logos">
+    <img
+      class="mitx_logo"
+      src="/static/images/logo-mitxonline.png"
+      alt="MITx"
+    />
+  </div>
+  <p>Courses for your program are now offered on MITx Online. To continue, please register a new account on MITx Online which will be linked to your existing MicroMasters account.</p>
```

Comment:
Any reason why we are using a generic "your program" here instead of specifying, i.e. 

```suggestion
  <p>Courses for {{program.title}} are now offered on MITx Online. To continue, please register a new account on MITx Online which will be linked to your existing MicroMasters account.</p>
```

Reply:
I think I was thinking that I couldn't rule out this being for more than one program, so i left the language generic on purpose.